### ISSUE TITLE
Bug: My device name is not visible on Firefox for Android

### ISSUE THREAD

**Issue by rogerm4242** [schlagmichdoch/PairDrop#73]
My device name is not visible on Firefox for Android. See screenshot. It is possible to work-around the issue by turning on "Desktop site", but that makes the site hard to read or use.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Samsung S21
 - OS: Android 13
 - Browser: Firefox for Android
 - Version: Firefox v 110.1.0

**Self-Hosted**
No

![Screenshot_20230310_105335_Firefox](https://user-images.githubusercontent.com/13337800/224364498-eca9ab6e-e3be-4ba0-9d73-4ef15c690168.jpg)

**Comment by schlagmichdoch** [comment]
That's some weird behaviour. Do you experience the same issue with other browsers / devices?
As you can see the circles in the background are at correct height so javascript calculates `document.documentElement.clientHeight` correctly. Apparantly, css `100vh` includes the navigation bar which is different to other browsers. I'll try to work around that behaviour and come back to you

**Comment by rogerm4242** [comment]
On DuckDuckGo's browser on the same phone, the page looks identical when loaded and the device name is hidden. However, on swiping the screen to scroll down, DuckDuckGo hides the top navigation bar and displays the device name. 

Firefox does not hide the top navigation bar, or scroll the page at all.

**Comment by schlagmichdoch** [comment]
Ok I played around a little with the min-height styles. Does everything look right on this instance? https://pairdrop-dev.onrender.com

**Comment by rogerm4242** [comment]
That looks good to me!

![Screenshot_20230310_130757_Firefox](https://user-images.githubusercontent.com/13337800/224391490-13ab82ed-33e7-4b4e-8e99-c97d1da1f521.jpg)

**Comment by schlagmichdoch** [comment]
Awesome! All other browser still look good too? Then I will merge this :)

**Comment by rogerm4242** [comment]
Yes, I tried the dev instance in DuckDuckGo browser also. It behaved the same as always, the name was hidden until I scrolled the page.

Thank you for the quick work!
